packages/select/src/vaadin-select-list-box.js

```diff
@@ -79,3 +79,5 @@ class SelectListBox extends ListMixin(ThemableMixin(DirMixin(ControllerMixin(Pol
 }
 
 defineCustomElement(SelectListBox);
+
+export { SelectListBox };
```

Comment:
The export is also missing in the Lit component.

Reply:
Thanks, added.